card/db/schema.rb

```diff
@@ -9,121 +9,120 @@
 # from scratch. The latter is a flawed and unsustainable approach (the more migrations
 # you'll amass, the slower it'll run and the greater likelihood for issues).
 #
-# It's strongly recommended that you check this file into your version control system.
+# It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20151105225559) do
+ActiveRecord::Schema.define(:version => 20141216053032) do
 
-  create_table "card_actions", force: :cascade do |t|
-    t.integer "card_id",         limit: 4
-    t.integer "card_act_id",     limit: 4
-    t.integer "super_action_id", limit: 4
-    t.integer "action_type",     limit: 4
+  create_table "card_actions", :force => true do |t|
+    t.integer "card_id"
+    t.integer "card_act_id"
+    t.integer "super_action_id"
+    t.integer "action_type"
     t.boolean "draft"
-    t.text    "comment",         limit: 65535
   end
 
-  add_index "card_actions", ["card_act_id"], name: "card_actions_card_act_id_index", using: :btree
-  add_index "card_actions", ["card_id"], name: "card_actions_card_id_index", using: :btree
+  add_index "card_actions", ["card_act_id"], :name => "card_actions_card_act_id_index"
+  add_index "card_actions", ["card_id"], :name => "card_actions_card_id_index"
 
-  create_table "card_acts", force: :cascade do |t|
-    t.integer  "card_id",    limit: 4
-    t.integer  "actor_id",   limit: 4
+  create_table "card_acts", :force => true do |t|
+    t.integer  "card_id"
+    t.integer  "actor_id"
     t.datetime "acted_at"
-    t.string   "ip_address", limit: 255
+    t.string   "ip_address"
   end
 
-  add_index "card_acts", ["actor_id"], name: "card_acts_actor_id_index", using: :btree
-  add_index "card_acts", ["card_id"], name: "card_acts_card_id_index", using: :btree
+  add_index "card_acts", ["actor_id"], :name => "card_acts_actor_id_index"
+  add_index "card_acts", ["card_id"], :name => "card_acts_card_id_index"
 
-  create_table "card_changes", force: :cascade do |t|
-    t.integer "card_action_id", limit: 4
-    t.integer "field",          limit: 4
-    t.text    "value",          limit: 16777215
+  create_table "card_changes", :force => true do |t|
+    t.integer "card_action_id"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.